README.md

```diff
@@ -1 +1,63 @@
+
+<b>[2]</b> <i>S. J. Taylor, A. Khan, L. K. Morse, A. Tolk, L. Yilmaz, J. Zander, and P. J. Mosterman, "Grand challenges for modeling and simulation: simulation everywhere — from cyberinfrastructure to clouds to citizens," Simulation: Transactions of the Society for Modeling and Simulation International, vol. 91, no. 7, pp. 648-665, 2015.</i>
+
+## Software Purpose
+
+In this project, we propose to implement a workflow-based approach meant to support the complete simulation lifecycle. Through this system, users will be able to submit geospatial data, convert them into spatial DEVS simulation models, and execute the simulation using the Cadmium simulator developed at the Advanced Real-Time Simulation Laboratory. Simulation results will then be optimized for web visualization in the DEVS WebViewer developed in the same laboratory.
+
+The purpose of this software is to address the two challenges previously identified. Automated generation of large-scale spatial simulation models from GIS data seeks to lessen the burden of modeling for modelers. The integration of several already existing applications (Model generator, Cadmium & DEVS WebViewer) through this software aims at democratizing the simulation lifecycle for non-expert users.
+
+## Software Requirements
+
+1. Users must be able to provide geospatial data as a source for the model generation workflow, in the following geospatial data formats: shapefile, CSV, GeoJSON. The data files provided should also contain non-spatial attributes.
+
+2. Users must be able to select the simulation model to be mapped onto data from a library of models.
+
+3. The system should be portable to any mainstream operating systems : Linux, MacOS, and Windows.
+
+4. The system will perform spatial analysis of the geospatial data provided to establish model parameters. The system must support common spatial analysis functions to establish neighborhood and parameters:
+	1. Topological relationships (intersection, contains, within, touches, etc.).
+	2. Geospatial statistics.
+	3. Network distance analysis.
+	4. Spatial buffers.
+
+5. Users must be able to provide a workflow file that contains the steps for automated model generation. The system will automatically execute the workflow to convert geospatial data into a Cadmium model configuration file. Through the workflow, the system will:
+	1. Map geospatial features onto selected parametric simulation models.
+	2. Establish neighborhoods or couplings through spatial analysis of the geospatial data.
+	3. Output a configuration file (json) properly formatted for Cadmium.
+
+6. The system will execute the simulation and return the results.
+
+7. The system must convert the Cadmium results into a common specification for web based visualization through an existing conversion web service.
```

Comment:
Are we able to use the web service here since Bruno is working on it?

Reply:
Yes. This is a different web service than the one I was talking about in the meeting.

Reply:
Sounds good!